DNN Platform/Library/Security/Permissions/Controls/PermissionsGrid.cs

```diff
@@ -1234,15 +1239,15 @@ private RoleInfo GetSelectedRole(int selectedRoleId)
                 role = new RoleInfo
                 {
                     RoleID = this.AllUsersRoleId,
-                    RoleName = Globals.glbRoleAllUsersName,

```

Comment:
I'm wondering if these changes are necessary, since `Localization.LocalizeRole` is called on line 819 when binding data into the grid.